crates/napi/src/next_api/project.rs

```diff
@@ -144,6 +144,9 @@ pub struct NapiProjectOptions {
 
     /// The browserslist query to use for targeting browsers.
     pub browserslist_query: String,
+
+    /// Whether mangling should be disabled
```

Comment:
This comment isn't useful. Either remove it, or add a better comment explaining what mangling is (shortening/renaming of local variable and function names).